### DISCUSSION TITLE
Can Neper be used to generate this model?

### DISCUSSION THREAD

**Discussion by Mulike-568** [neperfepx/neper#677] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/2659376/230397060-4697c9da-2767-42aa-b8fa-7fff4fdabe33.png)

**Comment by rquey** [comment, marked as answer]
Yes, it can. It is a typical Voronoi tessellation:
```
neper -T -n 4
```

This is a geo image, so:
```
neper -T -n 4 -format geo
gmsh n4-id1.geo
```

**Reply by Mulike-568** [reply]
Thank you, sir. But I don't need this code, these four grains are divided from the center alomost. And simply generating this model with your code is probably not what I wanted.
![99](https://user-images.githubusercontent.com/118234752/230398560-e315e044-f939-439b-89ce-99c2c3f9c3b0.jpg)
